stores/account.ts

```diff
@@ -508,6 +509,22 @@ export const useAccountStore = defineStore('account', () => {
     }
   }
 
+  const getStoredPlusRedirectInfo = () => {
+    try {
+      const stored = localStorage.getItem('plus_redirect_route')
+      return stored ? JSON.parse(stored) : null
+    }
+    catch (error) {
+      console.error('Failed to parse stored redirect info:', error)
+      clearStoredPlusRedirectInfo()
+      return null
+    }
+  }
+
+  const clearStoredPlusRedirectInfo = () => {
+    localStorage.removeItem('plus_redirect_route')
+  }
```

Comment:
if you move useStorage here, you can 
```suggestion
function clearPlusRedirectRoute() {
  plusRedirectRoute.value = null
}
```